book/src/SUMMARY.md

```diff
@@ -16,13 +16,15 @@
       - [CSINode Object](csi-node-object.md)
     - [Features](features.md)
         - [Secrets & Credentials](secrets-and-credentials.md)
-        - [Volume Snapshot & Restore](snapshot-restore-feature.md)
         - [Topology](topology.md)
         - [Raw Block Volume](raw-block.md)
         - [Skip Attach](skip-attach.md)
         - [Pod Info on Mount](pod-info.md)
         - [Ephemeral Local Volumes](ephemeral-local-volumes.md)
         - [Volume expansion](volume-expansion.md)
+        - [Data Sources](volume-datasources.md)
+	        - [Cloning](volume-cloning.md)
+            - [Volume Snapshot & Restore](snapshot-restore-feature.md)
```

Comment:
nit: indent is off

Reply:
ahhh, I thought that was a git diff artifact, but some tabs snuck in on line 25, fixed.